CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Add historical_balances function for wallet: listing historical balances for default address of the wallet.
 - Add toAddressId() method to Transaction class
+
+### Removed
```

Comment:
need readme updates https://github.com/coinbase/coinbase-sdk-nodejs?tab=readme-ov-file#you-can-import-the-sdk-as-follows-1